js/main.js

```diff
@@ -31,7 +31,7 @@ function handleSubmit(event) {
 
   $form.reset();
 
-  $ul.prepend(renderEntry('#entry-form'));
+  $ul.prepend(renderEntry('$newEntry'));
```

Comment:
Still not correct. What are the only two strings you are passing into `renderEntry`?